app/runtime/runtime/src/main/java/io/syndesis/integration/runtime/SyndesisRouteBuilder.java

```diff
@@ -56,6 +56,8 @@ protected SyndesisModel loadModel() throws Exception {
     @SuppressWarnings("unchecked")
     @Override
     public void configure() throws Exception {
+        getContext().addInterceptStrategy(new OutMessageCaptureInterceptStrategy());
+
```

Comment:
[CamelAutoConfiguration](https://github.com/apache/camel/blob/master/components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/CamelAutoConfiguration.java#L438-L448) should automatically register beans of type `InterceptStrategy` so it may be better to register `OutMessageCaptureInterceptStrategy` using spring-boot's `@Configuration` so you can conditionally enable it with a simple property (feature toggle)

Reply:
Will do